### PR TITLE
fix line height on advisory toggle

### PR DESCRIPTION
**Pull request by HananINouman**
### Summary
Fixes the line height in the advisory toggle item. 

### Screenshots
Screenshot 2023-04-13 at 3 21 23 PM

ticket: #427

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "rollup": "^2.70.1",
     "rollup-plugin-typescript2": "^0.31.2",
     "storybook-addon-designs": "^6.3.1",
+    "ts-node": "^10.9.1",
```
Comment (f1lander) [inline package.json:71]: why this is needed this pr?

**Review by f1lander — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (f1lander) [inline package-lock.json:1]: This files need to be removed, I guess you install by mistake using npm

File: yarn.lock
Comment (f1lander) [inline yarn.lock:1]: remove this change too, you need to remove the ts-node package using yarn remove -D `ts-node` the only change introduced for this PR needs to be the lineHeight

**Review by f1lander — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (HananINouman) [reply yarn.lock:1]: yes sorry by mistake

Comment (HananINouman) [reply yarn.lock:1]: Got the error again once I removed yarn/lock changes, Screenshot 2023-04-20 at 11 35 53 AM
Are we sure we need to install it globally but we added it to the launchpad package.json?

Comment (f1lander) [reply yarn.lock:1]: Add it as dev-dep

**Review by f1lander — APPROVED** [review]
just add ts-node as dev-dep

**Pull request merged**